packages/kit/src/utils/escape.js

```diff
@@ -59,8 +59,7 @@ function escape(str, dict, unicode_encoder) {
 
 /** @type {Record<string, string>} */
 const escape_html_attr_dict = {
-	'<': '&lt;',
-	'>': '&gt;',
+	'&': '&amp;',
```

Comment:
it's maybe worth a comment that this is for pulling the attribute out of the dom and the other one is for security